### PR TITLE
Fix some bugs on inMemoryDB

### PR DESCRIPTION
**Pull request by cp5555**

### CODE REVIEW

File: apache2/ag_mdb/ag_mdb_internal.h [hunk 1]
```diff
@@ -341,8 +341,8 @@ int Entry_setKeyValue(CPTR_VOID shm_base, PTR_OFFSET entry_id, const char* key,
  ** Define the name prefix of Read/Write lock on Windows
```
Comment (azhao155) [inline apache2/ag_mdb/ag_mdb_internal.h]: change comment? probably mentioned that in order to be global namespcace, global should be the prefix

File: apache2/persist_dbm.c [hunk 2]
```diff
@@ -609,7 +609,11 @@ static int collection_store_ex(int db_option, modsec_rec *msr, apr_table_t *col)
         if(ag_dbm == NULL) {
             //Create the DB
             root_dcfg = msr->dcfg1->root_config;
+#ifdef _WIN32           
+            dbm_filename = apr_pstrcat(root_dcfg->mp, "Global\\", root_dcfg->data_dir, "/", var_name->value, NULL);
```
Comment (azhao155) [inline apache2/persist_dbm.c:613]: so it will be global\/<dbname>, is this "\" needed?

Comment (cp5555) [reply apache2/persist_dbm.c]: Sure, I will change the comment

File: apache2/ag_mdb/ag_mdb_internal.h (hunk 1 above)
Comment (cp5555) [reply apache2/ag_mdb/ag_mdb_internal.h]: Sure, I will change the comment

File: apache2/persist_dbm.c (hunk 2 above)
Comment (cp5555) [reply apache2/persist_dbm.c:613]: There is no double quotes in the dbm_filename, the double quotes is just for string.

Comment (azhao155) [reply apache2/persist_dbm.c:613]: sorry i mean it's Global\/<dbname>

**Pull request merged**